cl/search/migrations/0089_opinionscited_depth.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.25 on 2019-12-28 22:56
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("search", "0088_add_harvard_opinions"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="opinionscited",
+            name="depth",
+            field=models.IntegerField(
+                db_index=True,
+                default=1,
+                help_text=b"The number of times the cited opinion was cited in the citing opinion",
+            ),
+        ),
+    ]
```

Comment:
Do you mind pulling this and your model.py tweaks since I landed them manually? (The migration was a lot more complex for reasons you probably don't care to know.)

Reply:
👍

Reply:
Fixed in a8d3cbb (reverting my commit). I then had to merge master to get the changes you landed. (I didn't want to rebase again now that we started the review.)